bridge/qb/client/functions.lua

```diff
@@ -121,80 +121,118 @@ end
 
 -- Getters
 
+---@param pool string
+---@param ignoreList? integer[]
+---@return integer[]
+local function getEntities(pool, ignoreList) -- luacheck: ignore
+    ignoreList = ignoreList or {}
+    local ents = GetGamePool(pool)
+    local entities = {}
+    local ignoreMap = {}
+    for i = 1, #ignoreList do
+        ignoreMap[ignoreList[i]] = true
+    end
+
+    for i = 1, #ents do
+        local entity = ents[i]
+        if not ignoreMap[entity] then
+            entities[#entities + 1] = entity
+        end
+    end
+    return entities
+end
+
 ---@deprecated use the GetGamePool('CVehicle') native directly
-functions.GetVehicles = function()
-    GetGamePool('CVehicle')
+functions.GetVehicles = function(ignoreList)
+    return getEntities('CVehicle', ignoreList)
 end
 
 ---@deprecated use the GetGamePool('CObject') native directly
-functions.GetObjects = function()
-    GetGamePool('CObject')
+functions.GetObjects = function(ignoreList)
+    return getEntities('CObject', ignoreList)
 end
 
 ---@deprecated use the GetActivePlayers native directly
-functions.GetPlayers = GetActivePlayers
-
----@deprecated use the GetGamePool('CPed') native directly
-functions.GetPeds = function(ignoreList)
+functions.GetPlayers = function(ignoreList)
     ignoreList = ignoreList or {}
-    local pedPool = GetGamePool('CPed')
-    local peds = {}
+    local plys = GetActivePlayers()
+    local players = {}
     local ignoreMap = {}
     for i = 1, #ignoreList do
         ignoreMap[ignoreList[i]] = true
     end
 
-    for i = 1, #pedPool do
-        local entity = pedPool[i]
-        if not ignoreMap[entity] then
-            peds[#peds + 1] = entity
+    for i = 1, #plys do
+        local player = plys[i]
+        if not ignoreMap[player] then
+            players[#players + 1] = player
         end
     end
-    return peds
+    return players
```

Comment:
Suggestion: it is possible to refactor `getEntities` to accept a pool of entities or a list of players instead of a pool name to use it here too.